wordpress-mobile/MediaPicker-iOS
# Optimize thumbnail fetch for albums

This PR optimizes the fetching of images by using opportunistic fetch of images and caching albums groups.

To test:
 - Open the media picker demo on a device with a big media collection ( > 5K) and with iCloud activated
 - Scroll the items up and down and see if the scrolling is quick and with no pauses.
 - Open the group picker and scroll up down the picker
 - Switch to the photos app and add image and add/remove albums
 - Switch back to the demo app and see if update happen correctly.
